.github/workflows/caa_build_and_push.yaml

```diff
@@ -29,7 +29,7 @@ on:
         required: false
         type: string
       git_ref:
-        default: 'main'
+        default: 'image-action'
```

Comment:
should revoke this change if we want to merge main?

Reply:
@liudalibj yes, I'm going to rollback it. thanks!